src/components/tutorialComponent/index.tsx

```diff
@@ -113,20 +118,22 @@ const TutorialDisplay: React.FC = (): JSX.Element => {
     <TutorialBG disp={tutorialDisplay}>
       <TutorialHeader>
         <StepButton
-          clickFunction={(): Promise<{ tutorialStep: number }> =>
-            setTutorialStep(tutorialStep - 1)
-          }
+          clickFunction={(): Promise<{ tutorialStep: number }> => {
+            localStorage.setItem(tutName, (tutStep - 1).toString());
+            return setTutorialStep(tutorialStep - 1);
+          }}
```

Comment:
The two click functions for the back and forth button can be confused into a single defined function and you can pass in a boolean or string to indicate to move forth or back a step. Very minor change but always think about avoiding repeating code.